main.go

```diff
@@ -500,6 +502,11 @@ func runCommand() int {
 		args.verbose = *verboseFlag
 		args.clangFlags = clangFlags
 		args.cppCode = *cppFlag
+
+	case "version":
+		fmt.Fprint(os.Stdout, version.Version())
```

Comment:
As I understood, empty test output happen because we use `os.Stdout` but in that case `stderr`. Please try and regenerate output.

Reply:
Made the changes. One question, can we just create a version command with `flag.NewFlagSet` method and use `SetOutput` method to set `io.Writer` type to be `stdout`.

Reply:
@UtkarshGupta-CS , feel free for experiments. 
In my point of view, we can finilaze that PR and you can add changes by your approach in next PR. Because now we have 1 bug - please run `UPDATE_SNAPSHOTS=true go test ./...` for updating file `.snapshots/c4go-TestCLI-func1-Version` and result must be:
```
(*bytes.Buffer)(development
)
```
Please clarify your point of view.

Reply:
And you can check result of CLI output - `go test -v -run=TestCLI`

Reply:
Done.